server/src/main/java/org/apache/iotdb/db/metadata/mtree/traverser/Traverser.java

```diff
@@ -179,4 +181,63 @@ public void setTemplateMap(Map<Integer, Template> templateMap, IMNodeFactory<N>
   public void setSkipPreDeletedSchema(boolean skipPreDeletedSchema) {
     this.skipPreDeletedSchema = skipPreDeletedSchema;
   }
+
+  @Override
+  protected IFAState tryGetNextState(
+      N node, IFAState sourceState, Map<String, IFATransition> preciseMatchTransitionMap) {
+    IFATransition transition;
+    IFAState state;
+    if (node.isMeasurement()) {
+      String alias = node.getAsMeasurementMNode().getAlias();
+      if (alias != null) {
+        transition = preciseMatchTransitionMap.get(alias);
+        if (transition != null) {
+          state = patternFA.getNextState(sourceState, transition);
+          if (state.isFinal()) {
+            return state;
+          }
+        }
+      }
+      transition = preciseMatchTransitionMap.get(node.getName());
+      if (transition != null) {
+        state = patternFA.getNextState(sourceState, transition);
+        if (state.isFinal()) {
+          return state;
+        }
+      }
+      return null;
+    }
+
+    transition = preciseMatchTransitionMap.get(node.getName());
+    if (transition == null) {
+      return null;
+    }
+    return patternFA.getNextState(sourceState, transition);
+  }
```

Comment:
It seems unnecessary. Remove it? The same as SchemaTreeMeasurementVisitor.

Reply:
Should be saved to process alias.